### PR TITLE
AFK Kicker stores AI cores

### PR DESCRIPTION
**Pull request by MisterLayne**
If you get disconnected for being AFK, your core also gets stored.

### CODE REVIEW

File: code/controllers/subsystems/inactivity.dm [hunk 1]
```diff
@@ -27,6 +27,12 @@ SUBSYSTEM_DEF(inactivity)
 
 					else if(issilicon(C.mob))
 						information = " while a silicon."
+						if(isAI(C.mob))
+							var/mob/living/silicon/ai/A = C.mob
+							empty_playable_ai_cores += new /obj/structure/AIcore/deactivated(A.loc)
+							global_announcer.autosay("[A] has been moved to intelligence storage.", "Artificial Intelligence Oversight")
+							A.clear_client()
+							information = " while an AI."
```
Comment (elgeonmb) [inline code/controllers/subsystems/inactivity.dm]: shouldve commented before this got merged; is this not going to wind up printing "[playername] was kicked for being AFK while a silicon while an AI?"

Comment (Anewbe) [reply code/controllers/subsystems/inactivity.dm]: Sets it equal to, doesn't add to.

**Comment by nerdass** [comment]
What behavior does this have if the player is just disconnected?

**Comment by Anewbe** [comment]
None, the AFK kicker only triggers if the player has been logged in and inactive for 45 minutes.

**Pull request merged**